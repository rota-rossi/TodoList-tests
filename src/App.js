import React from 'react';
import TodoList from './components/TodoList/TodoList';
import Title from './components/Title/Title';

export const App = () => (
  <div>
    <Title />
    <TodoList />
    <p>FOOTER!</p>
  </div>
);

export default App;
