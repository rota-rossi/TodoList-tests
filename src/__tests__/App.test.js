import React from 'react';
// import ShallowRenderer from 'react-test-renderer/shallow';
import renderer from 'react-test-renderer';
import App from '../App';

describe('App', () => {
  test('Should render properly', () => {
    const element = renderer.create(<App />).toJSON();
    // using the lines instead of the one above generates a shallow snapshot.
    // const renderer = new ShallowRenderer();
    // const element = renderer.render(<App />).getRenderOutput();

    expect(element).toMatchSnapshot();
  });
});
