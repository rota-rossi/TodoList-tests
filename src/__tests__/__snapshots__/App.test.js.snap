// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render properly 1`] = `
<div>
  <div
    className="title-area"
  >
    <h1
      className="title-text"
    >
      Todo List
    </h1>
  </div>
  <div
    className="todolist-area"
  >
    <ul>
      <li
        onClick={[Function]}
      >
        <div
          className="li-title"
        >
          Buy Milk
        </div>
        <div
          className="li-content"
        >
          2% fat
        </div>
      </li>
      <li
        onClick={[Function]}
      >
        <div
          className="li-title"
        >
          Take out the trash
        </div>
      </li>
      <li
        onClick={[Function]}
      >
        <div
          className="li-title"
        >
          Buy Costume
        </div>
      </li>
    </ul>
  </div>
  <p>
    FOOTER!
  </p>
</div>
`;
